magnet/src/main/java/magnet/internal/InstanceBucket.java

```diff
@@ -16,13 +16,12 @@
 
 package magnet.internal;
 
+import magnet.Scoping;
+import magnet.inspection.ScopeVisitor;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
+import java.util.*;
```

Comment:
Yeah, I'm using a macbook replacement (my one is in service because of that "battery recall" program of apple). That's why most of the settings are not tuned yet.